plugins/chat/assets/javascripts/discourse/chat-route-map.js

```diff
@@ -15,6 +15,8 @@ export default function () {
     this.route("channels");
     this.route("threads");
 
+    this.route("message-from-params", { path: "/new-message" });
```

Comment:
Did you consider:

```suggestion
    this.route("new-message", { path: "/new-message" });
```

Reply:
It felt a bit ambiguous without mentioning either url / link / params, but I'm okay with it so just went with `new-message` route name.